go.mod

```diff
@@ -6,9 +6,9 @@ toolchain go1.21.4
 
 require (
 	cloud.google.com/go/datastore v1.15.0
-	cloud.google.com/go/pubsub v1.33.0
-	cloud.google.com/go/secretmanager v1.11.2
-	cloud.google.com/go/storage v1.33.0
+	cloud.google.com/go/pubsub v1.36.1
```

Comment:
Why was the Google Cloud SDK updated? This was not in the table with the list of vulnerabilities

Reply:
this is required due to bumping the grpc package to the version without a security vulnerability. I just tried downgrading it back and it bumps us down for grpc pkg unfortunately

Reply:
See if using 1.60.1 requires bumping too, see other thread